src/drf_yasg/generators.py

```diff
@@ -34,15 +34,28 @@ def get_path_from_regex(self, path_regex):
             logger.warning("url pattern does not end in $ ('%s') - unexpected things might happen", path_regex)
         return self.unescape_path(super(EndpointEnumerator, self).get_path_from_regex(path_regex))
 
-    def should_include_endpoint(self, path, callback, app_name='', namespace='', url_name=None):
+    def get_version_regex_from_path_regex(self, path_regex, version_param):
```

Comment:
Will this function also work with Django 2.0 `path`? i.e. `path('api/<string:version>/one', include(...))`

Reply:
I'm unsure if `URLPathVersioning` works with 2.0 paths at all.

Reply:
Of course it does, and it would be considered broken if it didn't. It is evident from the Django and DRF docs that `path`, possibly with [custom converters](https://docs.djangoproject.com/en/2.2/topics/http/urls/#registering-custom-path-converters), is the preferred path templating mechanism going forward. 

`re_path`/`url` should only be used for compatibility, or when absolutely necessary.

Reply:
Updated for standard path converters - `string`, `int` and `slug` (well, it's weird to use other ones like `uuid` or `path` for versioning, and custom ones are not supported here (in `drf-yasg`) at all, as I see).